### PR TITLE
Make <a> TeX Purple

### PR DESCRIPTION
**Pull request by AlexJBanks**
make hyperlinks TeX purple so they stand out more on the website

### CODE REVIEW

File: css/clubs.scss [hunk 1]
```diff
@@ -49,6 +49,10 @@
     * {
       color: black;
     }
+
+    a {
+      color: #9d68dc;
```
Comment (Danyc0) [inline css/clubs.scss:54]: Was it an active decision to not use/define $TeX-Purple everywhere?

Comment (AlexJBanks) [reply css/clubs.scss:54]: I haven't found a way to globally define $TeX-Purple in SCSS. Happy to make these changes if you know how

Comment (LikkanChung) [reply css/clubs.scss:54]: Fine for now, but can raise an issue to refactor colour variables as tech debt

**Comment by LikkanChung** [comment]
Axe Accessibility Testing raises various issues across the site. Leave as tech debt and raise an issue

**Review by LikkanChung — APPROVED** [review]

**Pull request merged**